### PR TITLE
feat: sync meta discussion

### PR DESCRIPTION
**Pull request by vinckr**
Added a check to sync.sh to distinguish between ory main projects, 
who have their own discussion board and 
/meta/discussions that is for all other projects. 

Had to change the variables to $project (lowercase), since the substitution was case sensitive. 

Changed the variables in the contributing template accordingly. 

Also closes https://github.com/ory/meta/pull/76.

### CODE REVIEW

File: scripts/sync.sh [hunk 1]
```diff
@@ -44,7 +44,11 @@ function sync {
     # /bin/bash -c execute commands in " " in the new environment 
     # envsubst: replace all env variables found ($f) with the ones specified 2 steps before, pipe to sponge and write to file
     for f in $(find "$workdir/.github/ISSUE_TEMPLATE" "$workdir/.github/pull_request_template.md" "$workdir/CONTRIBUTING.md" "$workdir/SECURITY.md" "$workdir/CODE_OF_CONDUCT.md" "$workdir/LICENSE" -type f -print); do
-        env -i REPOSITORY="$project" PROJECT="$humanName" /bin/bash -c "envsubst < \"$f\" | sponge \"$f\""
+        if [[ "$project" = "hydra" || $project = "kratos" || $project = "oathkeeper" || $project = "keto" ]]; then
+        env -i discussions="$discussions" repository="$project" project="$humanName" /bin/bash -c "envsubst < \"$f\" | sponge \"$f\""
```
Comment (aeneasr) [inline scripts/sync.sh:48]: Please format and indent.

```suggestion
           env -i discussions="$discussions" repository="$project" project="$humanName" /bin/bash -c "envsubst < \"$f\" | sponge \"$f\""
```

File: scripts/sync.sh [hunk 2]
```diff
@@ -44,7 +44,11 @@ function sync {
...
+        else
+        env -i discussions="https://github.com/meta/discussions" repository="$project" project="$humanName" /bin/bash -c "envsubst < \"$f\" | sponge \"$f\""
```
Comment (aeneasr) [inline scripts/sync.sh:50]: Please format and indent.

```suggestion
            env -i discussions="https://github.com/meta/discussions" repository="$project" project="$humanName" /bin/bash -c "envsubst < \"$f\" | sponge \"$f\""
```

**Review by aeneasr — CHANGES_REQUESTED** [review]
This PR can not be accepted, it changes the environment variables from upper to lower case, but neglects to update all the references in the files updated by the `env` calls.

Please revert the lower case changes. Environment variables are upper case, not lower case.

**Comment by vinckr** [comment]
Changed the env variables to uppercase, now I know the convention 👍 .

I changed all variables in CONTRIBUTING.md, and also config.yml

In 
".github/ISSUE_TEMPLATE" 
".github/pull_request_template.md" 
"SECURITY.md" 
"CODE_OF_CONDUCT.md" 
"LICENSE"
I could not find any variables that are being replaced. 

Am I looking in the wrong spot (templates/repository/common)?

File: scripts/sync.sh [hunk 3]
```diff
@@ -44,7 +44,11 @@ function sync {
     # /bin/bash -c execute commands in " " in the new environment 
     # envsubst: replace all env variables found ($f) with the ones specified 2 steps before, pipe to sponge and write to file
     for f in $(find "$workdir/.github/ISSUE_TEMPLATE" "$workdir/.github/pull_request_template.md" "$workdir/CONTRIBUTING.md" "$workdir/SECURITY.md" "$workdir/CODE_OF_CONDUCT.md" "$workdir/LICENSE" -type f -print); do
-        env -i REPOSITORY="$project" PROJECT="$humanName" /bin/bash -c "envsubst < \"$f\" | sponge \"$f\""
+        if [[ "$project" = "ory/hydra" || $project = "ory/kratos" || $project = "ory/oathkeeper" || $project = "ory/keto" ]]; then
+          env -i Discussions="$discussions" Repository="$project" Project="$humanName" /bin/bash -c "envsubst < \"$f\" | sponge \"$f\""
```
Comment (aeneasr) [inline scripts/sync.sh:48]: As discussed, use UPPERCASE as it was before.

**Review by aeneasr — CHANGES_REQUESTED** [review]

**Comment by aeneasr** [comment]
Thank you, did you test the latest changes properly?

**Comment by vinckr** [comment]
I tested them as you suggested with 
f=path/to/file 
and then the 
```
if .... 
else
fi 
```
I couldnt test the whole script, but the parts that I touched worked.

**Comment by vinckr** [comment]
Missed one change that broke the git commands at the end of CONTRIBUTING.
All gucci now!

**Pull request merged**